lib/game.rb

```diff
@@ -6,30 +7,36 @@ class Game
   def initialize(input:, output:)
     @input = input
     @output = output
+    @grid = Grid.new
   end
 
   def play
     grid_view.render
-    get_player_input
+    take_player_turn
+    grid_view.render
   end
 
   private
 
-  attr_reader :input, :output
+  attr_reader :grid, :input, :output
 
-  def get_player_input
-    player_turn.get_coordinates
+  def take_player_turn
+    @grid = player_turn.mark_grid
```

Comment:
This method return value makes me nervous. When I look at what is passed into the initialize for the turn, and the name of the method, I expect it to mutate the grid state. So, when it is returned and important enough to hang on to, I worry. It makes me worry what would happens without that return value. 

I think this is just the nature of query/command mixing.